Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+                            @"NSNibBindingConnector", @"binding",
+                            @"NSWindowTemplate5", @"window",
+                            @"NSView", @"tableCellView",
+                            @"NSImage", @"image",
+                            @"IBUserDefinedRuntimeAttribute5", @"userDefinedRuntimeAttribute",
+                            nil];
+          RETAIN(XmltagToObjectClassCrossReference);
+
+          XmltagsNotStacked = [NSArray arrayWithObject: @"document"];
+          RETAIN(XmltagsNotStacked);
+
+          XmltagsToSkip = [NSArray arrayWithObject: @"dependencies"];
+          RETAIN(XmltagsToSkip);
+
+          ClassNamePrefixes = [NSArray arrayWithObjects: @"NS", @"IB", nil];
+          RETAIN(ClassNamePrefixes);
+
+          XmlReferenceAttributes = [NSArray arrayWithObjects: @"headerView", @"initialItem", nil];
+          RETAIN(XmlReferenceAttributes);
+
+          XmlConnectionRecordTags = [NSArray arrayWithObjects: @"action", @"outlet", @"binding", nil];
+          RETAIN(XmlConnectionRecordTags);
+
+          // These cross-reference from the OLD key to the NEW key that can be referenced and it's value
+          // or object returned verbatum.  IF an OLD XIB key does not exist and contains the 'NS' prefix
+          // the key processing will strip the 'NS' prefix, make the first letter lowercase then check
+          // whether that key exists and use it's presence during 'containsValueForKey:' processing, and
+          // use it's value for 'decodeXxxForKey:' processing.  So, the keys here should ONLY be those
+          // that cannot be generated autoamatically by this processing.
+          // (i.e. NSIsSeparator->isSeparatorItem, NSWindowStyleMask->styleMask, etc)
+          // Note, that unless the associated cross referenced key contains an aattribute that matches the
+          // original OLD key type you will need to potentially add a decoding method, and if so, the
+          // 'XmlKeyToDecoderSelectorMap' variable below should contain the key to it's associated decoding
+          // method for cross referencing...
+          XmlKeyMapTable = [NSDictionary dictionaryWithObjectsAndKeys:
+                                           @"isSeparatorItem", @"NSIsSeparator",
+                                           //@"systemMenu", @"NSName",
+                                           @"customClass", @"NSClassName",
+                                           @"catalog", @"NSCatalogName",
+                                           @"name" , @"NSColorName",
+                                           @"selectedItem", @"NSSelectedIndex",
+                                           @"pullsDown", @"NSPullDown",
+                                           @"prototype", @"NSProtoCell",
+                                           @"metaFont", @"IBIsSystemFont",
+                                           //@"headerView", @"NSHeaderClipView",
```

Comment:
Do we know why `headerView`, `minColumnWidth` et al are commented out?

Reply:
I could just remove two of the commented out values that where already handle differently, but there are still a few remaining. Just like you I don't like commented out code. Here we would need an example of a XIB file with that element to write code that correctly handles this. Until then this is a reminder that something is missing.